Modules/SubModule_Population/xASL_wrp_CreateBiasfield.m

```diff
@@ -23,10 +23,27 @@ function xASL_wrp_CreateBiasfield(x)
 %              PM: add normalization of between-subjects SD as well.

```

Comment:
I'm really confused. Why do we see all the edits here that we've done in a different issue? 
It looks like you have rebase this branch. In general, it is a really really bad idea to rebase a branch anytime before the PR is approved. If necessary to rebase, this has to be done in another way. Cause it makes then really complicated to asses the PR. It's OKish in this case, but otherwise we shouldn't do that?

Or is there another explanation why do I see edits here from another issue?

Reply:
No clue!